test/studies/shootout/thread-ring/README

```diff
@@ -1,26 +0,0 @@
-This directory showcases different versions of the Computer Language Benchmark
-Game problem "thread-ring".  The significant difference between the programs
-is each version uses a different task parallelism idiom: the current preferred
-version utilizes a coforall loop to start the individual threads, and within
-that loop uses a begin statement to fire off the tasks.  This was discovered to
-be faster than utilizing solely coforall (an alternate version also present in
-this directory), and as expected was faster than a version that utilizes only
-begin statements within a for loop (also present here).  All three versions are
-retained within the subdirectory lydia in case in the future the performance of
-one or all of these strategies alters for better or for worse.
-
-Other changes tried: altering chpl_task_alloc calls in the c code to
-chpl_malloc calls caused little to no improvement on its own, but when combined
-with deletions of extraneous _ic_these_DefaultRectangularDom_1_int64_t_F alloc's
-and free's caused a fair improvement, although not as much as we would like.
-Utilizing writeXF instead of writeEF also seemed to give a slight improvement.
-
-Skipping testing under valgrind because the current settings do not allow enough
-threads to complete even under the (comparatively) low number of threads for
-that sample size
```

Comment:
I initially just started updating the file, but decided to just remove it since it's largely out of date and I don't think provides much value. For instance: 

 - the coforall+begin version is no longer the fastest 
 - chpl_task_alloc isn't a thing anymore 
 - there are no instances of _ic_these in the generated code anymore
 - the timings for qthreads/muxed are out of date

The only current info is about skipping under valgrind, and I don't think that's useful enough info to warrant keeping the readme around